ui/apps/platform/package.json

```diff
@@ -14,7 +14,8 @@
     "license": "UNLICENSED",
     "dependencies": {
         "@apollo/client": "^3.6.3",
-        "@lifeomic/axios-fetch": "^3.0.1",
+        "@lifeomic/axios-fetch": "^3.1.0",
```

Comment:
Copied the instructions from https://github.com/stackrox/stackrox/pull/14661#issuecomment-2741165805. There was a merge conflict in the process. We had to upgrade the version of this dependency. Not sure why though.